Formula/libgcrypt.rb

```diff
@@ -1,9 +1,9 @@
 class Libgcrypt < Formula
   desc "Cryptographic library based on the code from GnuPG"
   homepage "https://gnupg.org/related_software/libgcrypt/"
-  url "https://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.8.6.tar.bz2"
-  sha256 "0cba2700617b99fc33864a0c16b1fa7fdf9781d9ed3509f5d767178e5fd7b975"
-  license "GPL-2.0"
+  url "https://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.8.7.tar.bz2"
+  sha256 "03b70f028299561b7034b8966d7dd77ef16ed139c43440925fe8782561974748"
+  license "GPL-2.0-only"
```

Comment:
```
/* gcrypt.h -  GNU Cryptographic Library Interface              -*- c -*-
 * Copyright (C) 1998-2017 Free Software Foundation, Inc.
 * Copyright (C) 2012-2017 g10 Code GmbH
 *
 * This file is part of Libgcrypt.
 *
 * Libgcrypt is free software; you can redistribute it and/or modify
 * it under the terms of the GNU Lesser General Public License as
 * published by the Free Software Foundation; either version 2.1 of
 * the License, or (at your option) any later version.
 *
 * Libgcrypt is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this program; if not, see <http://www.gnu.org/licenses/>.
 *
 * File: @configure_input@
 */
```

```
    License
    -------

    The library is distributed under the terms of the GNU Lesser
    General Public License (LGPL); see the file COPYING.LIB for the
    actual terms.

    The helper programs as well as the documentation are distributed
    under the terms of the GNU General Public License (GPL); see the
    file COPYING for the actual terms.

    The file LICENSES has notices about contributions that require
    that these additional notices are distributed.
```

Reply:
Strange -- the `COPYING` file inside `libgcrypt-1.8.7.tar.bz2` is just the standard GPLV2 one:
```
		    GNU GENERAL PUBLIC LICENSE
		       Version 2, June 1991

 Copyright (C) 1989, 1991 Free Software Foundation, Inc.
     59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 Everyone is permitted to copy and distribute verbatim copies
 of this license document, but changing it is not allowed.
```

Reply:
that is the file without header.

GPL license needs header declaration either in the README or source files. The COPYING or LICENSE file alone won't tell the full story.

Reply:
Ah, ok -- thanks for the explanation @chenrui333.

So this one should be `LGPL-2.1-or-later`?